### PR TITLE
Update ClickHouse docs crawler config

### PR DESCRIPTION
**Pull request by blinkov**
# Pull request motivation

1. Documentation moved to new domain
2. Non-default version docs are hidden from normal search engines, but also need some in-site search

### CODE REVIEW

File: configs/yandex_clickhouse.json [hunk 1]
```diff
@@ -20,16 +20,29 @@
...
   },
   "min_indexed_level": 2,
   "conversation_id": [
     "1058861561"
   ],
-  "nb_hits": 50105
-}
\ No newline at end of file
+  "nb_hits": 50105,
+  "ignoreRobotsTxtRules": true
```
Comment (s-pace) [inline configs/yandex_clickhouse.json]: What is the use of this parameter? It is not supported from our scraper IIRC

Comment (blinkov) [reply configs/yandex_clickhouse.json]: @s-pace I've found it over here: https://www.algolia.com/doc/api-reference/crawler/configuration/ignore-robots-txt-rules/

Comment (s-pace) [reply configs/yandex_clickhouse.json]: Thanks for the details. The URL you are referring to is the enterprise crawler offered by Algolia. This is another tool completely different from the DocSearch Crawler. [You can find out more information on this page](https://www.algolia.com/products/crawler/).

The documentation related to the DocSearch crawler is available [from this dedicated documentation](https://community.algolia.com/docsearch/config-file.html). The DocSearch crawler does not rely on the robots.txt so no worries.

Comment (s-pace) [reply configs/yandex_clickhouse.json]: Please remove this attribute from the configuration so that I can deploy.

Comment (blinkov) [reply configs/yandex_clickhouse.json]: @s-pace done, thanks

**Pull request merged**